pkg/uploader/kopia/progress.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright The Velero Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kopia
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/vmware-tanzu/velero/pkg/uploader"
+)
+
```

Comment:
Could you add some comments to the definition of objects and functions to help better understand?

Reply:
added